ext/native-decls/MumbleDoesChannelExist.md

```diff
@@ -0,0 +1,14 @@
+---
+ns: CFX
+apiset: client
+---
+## MUMBLE_DOES_CHANNEL_EXIST
+
+```c
+void MUMBLE_DOES_CHANNEL_EXIST(int channel);
```

Comment:
Please declare the return type correctly: `BOOL`.

Reply:
Oops, that was an oversight on my part, fixed it with latest commit